hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/resolver/backend/HttpBackendResolver.java

```diff
@@ -64,6 +66,11 @@
       if (httpOperation != null) {
         enrichedAttributes.put(BACKEND_OPERATION_ATTR, AttributeValueCreator.create(httpOperation));
       }
+      Optional<String> httpsDestination = HttpSemanticConventionUtils.getOtelHttpTarget(event);
```

Comment:
https? Is this only valid of HTTPs protocol?

Reply:
typo.